### PR TITLE
UnixReady::priority conflicts with Ready::writable (on Linux at least)

### LINKED ISSUE #896: UnixReady::priority conflicts with Ready::writable (on Linux at least)
On Linux, at least, I have ``Ready::from(UnixReady::priority()).is_writable() = true``

This is problematic, because you might be interested to watch for priority/exceptional condition, but not for writability.

### PR DESCRIPTION
**Pull request by elmarco**
fixes #896

### CODE REVIEW

File: src/sys/unix/ready.rs [hunk 1]
```diff
@@ -111,11 +111,24 @@ const LIO: usize   = 0b00_0000;
```
Comment (Thomasdezeeuw) [inline src/sys/unix/ready.rs]: While you're here can you remove the double empty line?

**Comment by carllerche** [comment]
@Thomasdezeeuw feel free to merge whenever.

**Comment by Thomasdezeeuw** [comment]
Will merge once CI is ready.

**Comment by Thomasdezeeuw** [comment]
Thanks @elmarco.

**Pull request merged**